frontend/app/package.json

```diff
@@ -107,8 +107,8 @@
       "no-unused-vars": "warn",
       "no-var": "warn",
       "no-extra-boolean-cast": "warn",
-      "no-useless-escape": "warn",
-      "no-prototype-builtins": "warn",
+      "no-useless-escape": "off",
+      "no-prototype-builtins": "off",
```

Comment:
Instead of turning these off in the package.json can we make an inline ignore in the code instead?

it would allow us to keep these warns but ignore them for the changes in this PR

Reply:
oh I guess you didn't add any new eslint errors, I'm going to change these to mark as warnings up move up the warn count